packages/encoder/test/address.test.ts

```diff
@@ -173,6 +173,18 @@ describe("Encoding", () => {
       );
     });
 
+    it("Encodes callable objects with an address field", async () => {
+      let input: any = () => undefined;
+      input.address = "0x10ca7e901d10CA7E901D10Ca7e901D10CA7e901D";
+      input.garbate = "garbage";
```

Comment:
@haltman-at is `garbate` a typo?

Reply:
Oops, thanks, it is!  Fortunately it's a deliberate garbage field so it's irrelevant. :)